### DISCUSSION TITLE
Discussion Bounties via One-Time GitHub Sponsor?

### DISCUSSION THREAD

**Discussion by HDVinnie** [community/community#11144] [Discussions]
Maybe a long stretch but might be cool and help FOSS devs by adding a feature to add a bounty via GitHub Sponsors one-time donation to say a Features discussion. 

For example:

https://github.com/HDInnovations/UNIT3D-Community-Edition/discussions/1740

This user may want to pay a bounty of $25 to have this added.

**Comment by HDVinnie** [comment]
Any feedback?

**Reply by Malvoz** [reply]
I believe this is duplicate of https://github.com/github/feedback/discussions/4517.
